### PR TITLE
Add support for Pagure

### PR DESCRIPTION
**Pull request by felixonmars**

### CODE REVIEW

File: nvchecker_source/pagure.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+from nvchecker.api import (
+  VersionResult, Entry, AsyncCache, KeyManager,
+)
+
+PAGURE_URL = 'https://%s/api/0/%s/git/tags'
+
+logger = structlog.get_logger(logger_name=__name__)
+
+async def get_version(name, conf, **kwargs):
+  return await get_version_real(name, conf, **kwargs)
```
Comment (lilydjwg) [inline nvchecker_source/pagure.py:17]: This indirection isn't needed.

File: nvchecker_source/pagure.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+async def get_version_real(
+  name: str, conf: Entry, *,
+  cache: AsyncCache, keymanager: KeyManager,
+  **kwargs,
+) -> VersionResult:
+  repo = conf['pagure']
+  host = conf.get('host', "pagure.io")
+
+  url = PAGURE_URL % (host, repo)
+  print(url)
```
Comment (lilydjwg) [inline nvchecker_source/pagure.py:28]: This should be removed.

File: README.rst [hunk 3]
```diff
@@ -633,6 +634,20 @@ This enables you to track updates of macOS applications which using `Sparkle fra
 sparkle
   The url of the sparkle appcast.
 
+Check Pagure
+------------
+::
+
+  source = "pagure"
+
+This enables you to check updates from `Pagure <https://pagure.io>`_.
```
Comment (lilydjwg) [inline README.rst:643]: The docs should say tags are returned and support `list options`.

**Review by lilydjwg — CHANGES_REQUESTED** [review]

**Comment by felixonmars** [comment]
All addressed.

**Comment by lilydjwg** [comment]
Thanks1

**Pull request merged**